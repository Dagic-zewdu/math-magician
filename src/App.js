import React from 'react';
import { BrowserRouter, Route, Router } from 'react-router-dom';
import CalcContainer from './components/calculator/calc.container';
import Home from './components/home/home';
import Navbar from './components/layout/navbar';
import Quote from './components/quote/quote';

const App = () => (
  <BrowserRouter>
    <Navbar />
    <Router>
      <Route exact path="/" element={<Home />} />
      <Route path="/calculator" element={<CalcContainer />} />
      <Route path="/quote" element={<Quote />} />
    </Router>
  </BrowserRouter>
);

export default App;
