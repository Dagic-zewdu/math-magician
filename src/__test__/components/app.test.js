import React from 'react';
import renderer from 'react-test-renderer';
import App from '../../App';

describe('should render without problems', () => {
  it('should render the routes properly', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
