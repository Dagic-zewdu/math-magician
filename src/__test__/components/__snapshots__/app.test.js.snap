// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without problems should render the calculator 1`] = `
Array [
  <header
    className="sc-dkPtRN lfxPxL"
  >
    <a
      className="sc-hKwDye cLeryG"
      href="/"
      onClick={[Function]}
    >
      Math Magician
    </a>
    <div
      className="sc-eCImPb bElXed"
    >
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
      >
        Home |
      </a>
      <a
        className=""
        href="/calculator"
        onClick={[Function]}
      >
        Calculator |
      </a>
      <a
        className=""
        href="/quote"
        onClick={[Function]}
      >
        Quote 
      </a>
    </div>
  </header>,
  <div
    className="sc-gsDKAQ giimbc"
  >
    <p>
      Lorem ipsum dolor sit, amet consectetur adipisicing elit. Odit quas consequuntur itaque odio maiores rem tempora laudantium cum minima! Sit rerum tenetur quibusdam architecto, similique veniam eaque quas quis praesentium odio tempore molestias unde, atque beatae quod. Eveniet, qui hic voluptatum amet nesciunt ut est commodi praesentium quod aliquam nisi fugiat excepturi quaerat officia suscipit accusantium, molestias assumenda? Excepturi vero, natus fugit, possimus corporis quasi repellendus amet voluptatibus alias enim consequatur dignissimos quam quos aperiam sed esse deleniti sit itaque culpa! Cum deserunt illum, consequatur dolores, voluptas ut voluptatem placeat pariatur officia obcaecati velit accusamus. Repellat nesciunt natus iure maiores praesentium? Providen t totam maiores explicabo temporibus earum placeat inventore quos nesciunt repellendus minima quibusdam quod dicta, asperiores fugit reprehenderit amet! Doloribus vero et perferendis fugit molestias vel magnam doloremque impedit nisi hic quas officia earum saepe, libero ea maxime blanditiis dolorum labore. Debitis et eos, repellat accusantium, laboriosam doloribus rem nobis sequi vero sint placeat labore facilis, sapiente dignissimos. Atque deserunt, optio officiis explicabo impedit earum sequi vero ab exercitationem. Et laudantium obcaecati beatae fuga ad, eligendi debitis iure amet. Iusto consequatur eum accusamus eligendi, laboriosam explicabo ratione assumenda illo impedit tempora adipisci mollitia, eos laborum recusandae accusantium illum dolore!
    </p>
  </div>,
]
`;

exports[`should render without problems should render the routes properly 1`] = `
Array [
  <header
    className="sc-dkPtRN lfxPxL"
  >
    <a
      className="sc-hKwDye cLeryG"
      href="/"
      onClick={[Function]}
    >
      Math Magician
    </a>
    <div
      className="sc-eCImPb bElXed"
    >
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
      >
        Home |
      </a>
      <a
        className=""
        href="/calculator"
        onClick={[Function]}
      >
        Calculator |
      </a>
      <a
        className=""
        href="/quote"
        onClick={[Function]}
      >
        Quote 
      </a>
    </div>
  </header>,
  <div
    className="sc-gsDKAQ giimbc"
  >
    <p>
      Lorem ipsum dolor sit, amet consectetur adipisicing elit. Odit quas consequuntur itaque odio maiores rem tempora laudantium cum minima! Sit rerum tenetur quibusdam architecto, similique veniam eaque quas quis praesentium odio tempore molestias unde, atque beatae quod. Eveniet, qui hic voluptatum amet nesciunt ut est commodi praesentium quod aliquam nisi fugiat excepturi quaerat officia suscipit accusantium, molestias assumenda? Excepturi vero, natus fugit, possimus corporis quasi repellendus amet voluptatibus alias enim consequatur dignissimos quam quos aperiam sed esse deleniti sit itaque culpa! Cum deserunt illum, consequatur dolores, voluptas ut voluptatem placeat pariatur officia obcaecati velit accusamus. Repellat nesciunt natus iure maiores praesentium? Providen t totam maiores explicabo temporibus earum placeat inventore quos nesciunt repellendus minima quibusdam quod dicta, asperiores fugit reprehenderit amet! Doloribus vero et perferendis fugit molestias vel magnam doloremque impedit nisi hic quas officia earum saepe, libero ea maxime blanditiis dolorum labore. Debitis et eos, repellat accusantium, laboriosam doloribus rem nobis sequi vero sint placeat labore facilis, sapiente dignissimos. Atque deserunt, optio officiis explicabo impedit earum sequi vero ab exercitationem. Et laudantium obcaecati beatae fuga ad, eligendi debitis iure amet. Iusto consequatur eum accusamus eligendi, laboriosam explicabo ratione assumenda illo impedit tempora adipisci mollitia, eos laborum recusandae accusantium illum dolore!
    </p>
  </div>,
]
`;
